pghoard/rohmu/object_storage/azure.py

```diff
@@ -14,13 +14,16 @@
 
 
 class AzureTransfer(BaseTransfer):
-    def __init__(self, account_name, account_key, bucket_name, prefix=None):
+    def __init__(self, account_name, account_key, bucket_name, prefix=None,
+                 azure_cloud=None):
         prefix = "{}".format(prefix.lstrip("/") if prefix else "")
         super().__init__(prefix=prefix)
         self.account_name = account_name
         self.account_key = account_key
         self.container_name = bucket_name
-        self.conn = BlockBlobService(account_name=self.account_name, account_key=self.account_key)
+        endpoint_suffix = "core.cloudapi.de" if azure_cloud == "germany" else None
```

Comment:
`README.rst` says `public` is the default, but there's no validation for `azure_cloud`.  How about a dict mapping cloud names to endpoints or something like that to catch errors.